apollo-router/src/plugins/telemetry/config_new/events.rs

```diff
@@ -690,14 +641,6 @@ pub(crate) enum EventOn {
 }
 
 pub(crate) struct CustomEvent<Request, Response, EventResponse, A, T>
-where
-    A: Selectors<Request, Response, EventResponse> + Default,
-    T: Selector<Request = Request, Response = Response> + Debug,
-{
-    pub(super) inner: Mutex<CustomEventInner<Request, Response, EventResponse, A, T>>,
-}
-
-pub(super) struct CustomEventInner<Request, Response, EventResponse, A, T>
```

Comment:
I removed "inner" since we no longer need Mutex, we maintain a single copy and mutate it directly.